src/Pages/EmailList/email-list.css

```diff
@@ -0,0 +1,45 @@
+.email-list {
+    background-color: #ecebeb;
+    height: 100vh;
+}
+
+#email-filter-button {
+    background-color: rgb(2, 107, 126) !important;
+    color: white !important;
+}
+
+#email-copy-button {
+    border-color: rgb(2, 107, 126) !important;
+    color: rgb(2, 107, 126) !important;
+}
+
+#email-copy-button:hover {
+    background-color: rgb(2, 107, 126) !important;
+    color: white !important;
+}
+
+#email-list-filter {
+    margin: 0px 15%;
+}
+
+#email-list-filter button {
+    margin-right: 8px;
+}
+
+#emails {
```

Comment:
im being nitpicky here but can you make css id names singular? and give it a better name than `email`